### DISCUSSION TITLE
Unable to install Aether on Ubuntu

### DISCUSSION THREAD

**Discussion by abuammarazmi** [KAHR-Alpha/Aether#28] [Q&A] [answered]
I have been trying to install and run Aether on Ubunut but it seems like I'm stuck. I tried installing file Aether_GUI after extracting from tar.gz as well as .7z files. It doesn't seem to work. I'm running it using wine on my device my would like to know a proper method to install or run it on ubuntu.

**Comment by KAHR-Alpha** [comment]
Hello, which version are you trying yo run?

The version found in the tar.gz and .7z files should have been built natively for ubuntu by GitHub. As such, you shouldn't need to use wine.

Maybe you're missing some of the dependencies?

**Reply by abuammarazmi** [reply]
I'm running ubuntu 22.04 and Aether's latest version 0.13. I did check again, all the dependencies are present. I don't know why I'm not able to run it. So for the .7z or tar.gz files, we just extract it and in the bin folder you run "Aether_GUI" right?

**Reply by KAHR-Alpha** [reply]
Technically it should work like that yes.
With that said, I booted up a Ubuntu machine that was sleeping, and tried launching Aether_GUI from the terminal.
I did get an error "libfftw3.so.3.6.9: cannot open shared object file: No such file or directory"

Now the fun part is that this lib doesn't seem to exist except for Alpine Linux.
I need to investigate the issue deeper.

In the meantime, have you tried building from source, or running the windows version through Wine (that used to work a long, long time ago)?

**Reply by abuammarazmi** [reply]
For me whenever I click on Aether_GUI, it doesn't show anything at all! I tried many times. I'll try to build from source and will see. I have been using the windows version through wine and it seems to have no problem. It's just that I don't have windows on my machine so I wanted to bring to your attention this problem. 😊

**Reply by KAHR-Alpha** [reply]
Yeah when you just click it the error message is silent, that's why I ran it through the terminal.

But thank you very much for the report, I didn't notice the issue as I work on Windows nearly all the time.
May I ask which part of Aether you're using?

**Comment by KAHR-Alpha** [comment, marked as answer]
Alright, I believe the latest release 0.14.1 could be fixing the issue.
It does start on my Ubuntu machine now.
